src/installer/pkg/projects/netcoreapp/pkg/crossgenRIDs.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+
+  <!-- Crossgen2 currently only builds for a subset of RIDs. -->
+  <PropertyGroup>
+    <BuildOnUnknownPlatforms>false</BuildOnUnknownPlatforms>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageRID Condition="'$(PackageRID)' == ''">$(OutputRid)</PackageRID>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <OfficialBuildRID Include="linux-x64" />
```

Comment:
This produced a linux-x64 crossgen pack in my local builds, but I don't see one in the mock official build I ran. Looking into it.